name.abuchen.portfolio/src/name/abuchen/portfolio/online/impl/MyDividends24Uploader.java

```diff
@@ -0,0 +1,108 @@
+package name.abuchen.portfolio.online.impl;
```

Comment:
Maybe this is a better solution:
[MyDividends24Uploader.java.txt](https://github.com/buchen/portfolio/files/11238578/MyDividends24Uploader.java.txt)

If you have time, you can check this out.

Reply:
I tested it, with your implementation, the upload doesn't work. 
Is this the only problem why the Build didn't succeed?